packages/block-library/src/video/edit.js

```diff
@@ -262,8 +261,7 @@ function VideoEdit( {
 							</p>
 							{ !! poster && (
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
This can be seen while you add `core/video`, and check on inspector control while you add video poster image and after adding check for the `remove` button.

| Before | After     |
| :-------- | :------- |
| <img width="215" alt="video-poster-before-remove" src="https://github.com/user-attachments/assets/c82dcec1-da89-4eea-8a02-caf0f07336ab"> | <img width="215" alt="video-poster-after-remove" src="https://github.com/user-attachments/assets/19a2ba9f-4d0d-45a6-9d9b-d3b7324c76da"> |